sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/sampleoverview/samplelist/ProjectSamplesView.groovy

```diff
@@ -29,10 +27,11 @@ class ProjectSamplesView extends VerticalLayout {
 
     private void initLayout() {
         this.setMargin(false)
-        this.setSizeUndefined()
+        this.setSizeFull()
```

Comment:
I think we need the undefined size to enable scrolling of the child elements. This is an unfortunate vaadin fail.

Reply:
did you test it? I tried it out and I had no issues 🤔